src/components/select/select.jsx

```diff
@@ -47,7 +47,7 @@ const Select = forwardRef(function Select(props, ref) {
   useDropdownClose(wrapperRef, showOptions, handleDropdownClose);
 
   const clear = () => {
-    setValue(undefined);
+    setValue(null);
     setSearchValue(undefined);
```

Comment:
Should this also go to null? As a reviewer, it's very weird to me that sometimes we use `undefined` and other times we use `null`

Reply:
This is just text used internally in the component, so it doesn't actually matter which it is... and I opted to not change it. 😅